helix-core/src/indent.rs

```diff
@@ -281,27 +281,32 @@ impl Indentation {
 }
 
 /// An indent definition which corresponds to a capture from the indent query
+#[derive(Debug)]
 struct IndentCapture {
     capture_type: IndentCaptureType,
     scope: IndentScope,
 }
-#[derive(Clone, Copy)]
+#[derive(Debug, Clone, Copy, PartialEq)]
 enum IndentCaptureType {
     Indent,
+    IndentAlways,
     Outdent,
+    OutdentAlways,
 }
 impl IndentCaptureType {
     fn default_scope(&self) -> IndentScope {
         match self {
             IndentCaptureType::Indent => IndentScope::Tail,
+            IndentCaptureType::IndentAlways => IndentScope::Tail,
```

Comment:
These match arms van be merged with OR patterns as above (there is a clippynlint for this, I thought it was on by default maybe we should turn it on)